clusters/hive/pools/cvp/cvp-ocp-4-9-amd64-aws-us-west-2_clusterpool.yaml

```diff
@@ -1,22 +1,25 @@
 apiVersion: hive.openshift.io/v1
 kind: ClusterPool
 metadata:
-  name: cvp-ocp-4-9-amd64-aws-us-west-2
-  namespace: cvp-cluster-pool
+  creationTimestamp: null
```

Comment:
Do we need `creationTimestamp`?

Reply:
No, but the `clusterimageset-updater` loads the file as a struct and then remarshals it. A side effect is `creationTimestamp` being set to `null` and all unset fields getting default values (which is why the `int` status fields got set to `0` as well). I ran the `clusterimageset-updater` tool on the repo after adding the labels, so those changes are in this PR.